genie-web/src/integTest/java/com/netflix/genie/web/apis/rest/v3/controllers/RestControllerIntegrationTestBase.java

```diff
@@ -650,4 +660,29 @@ public void describeTo(final Description description) {
             description.appendText(this.mismatchDescription);
         }
     }
+
+    @Configuration
+    static class TestConfiguration {
+
+        private final TemporaryFolder attachmentsDirectory;
```

Comment:
I believe these tests have all been moved to JUnit 5. With that in mind this is from Junit 4 so should move to [one of the mechanisms](https://www.baeldung.com/junit-5-temporary-directory) for using JUnit 5 temp dir so we can eventually rip out junit 4.

Reply:
Ah dang i got confused, i should use @TempDir